docs/book/migration/to-3-0.md

```diff
@@ -0,0 +1,22 @@
+# Migration guide
+
+Migration guide for Laminas Cache version 3.0.0.
+
+## New Features
+
+- Each cache adapter has its [own package](#satellite-packages).
+
+## Breaking Changes
+
+- Removal of support for `laminas/laminas-servicemanager` and the related files
+    - `PatternPluginManagerV2Polyfill`
+    - `PatternPluginManagerV3Polyfill`
+    - `PatternPluginManagerTrait`
+
+## Satellite packages
+
+Starting with laminas-cache v3, we are introducing satellite packages for each cache backend.
+
+In order to make this package work, you have to specify at least **one** satellite package.
+
+A list of available cache adapters can be found here. `TODO: add link to documentation`.
```

Comment:
I think this information is relevant for every user of the component and should not be hidden in a migration guide. I suggest to add this information to the introduction and update the link above. So nothing is lost and everyone can find this information and list.